src/middleware/verifyToken.js

```diff
@@ -0,0 +1,23 @@
+import jwt from 'jsonwebtoken';
+
+/**
+ * @method verifyToken
+ * - Gets token from req.params and verifies from requests
+ */
+
+const verifyToken = (req, res, next) => {
+  const token = req.headers.authorization;
+  if (!token || typeof token === 'undefined') {
```

Comment:
do we need this part `typeof token === 'undefined'`? coz I'd expect  `!token` to evaluate to true for undefined